sources/academy/tutorials/api/run_actor_and_retrieve_data_via_api.md

```diff
@@ -102,15 +102,15 @@ import got from 'got';
 
 // Specify your API token
 // (find it at https://console.apify.com/account#/integrations)
-const myToken = <YOUR_APIFY_TOKEN>;
+const myToken = YOUR_APIFY_TOKEN;
```

Comment:
maybe better to keep that as string instead of some undefined var? something like this?

```suggestion
const myToken = '<YOUR_APIFY_TOKEN>';
```